source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Vulnerability Disclosure and security.txt
+last_reviewed_on: 2021-04-09
+review_in: 6 months
+---
+# <%= current_page.data.title %>
+
+## Vulnerability Disclosure
+
+In the Cabinet Office, including GDS, the [CDIO cyber security team] run a
+vulnerability disclosure programme with HackerOne and a third party to triage
```

Comment:
Should we name the third party?

Reply:
HackerOne is named as it's the entry point and listed in the public security.txt https://github.com/alphagov/security.txt/blob/5c51abbb6ff302c013340b8f589b350c2cdafe9d/security.txt#L3 I was thinking not to, but NCSC list them here: https://www.ncsc.gov.uk/blog-post/ncsc-vulnerability-disclosure-co-ordination so yeah, I'll update!